### PR TITLE
tree-from-tags: init at 1.1

### PR DESCRIPTION
**Pull request by livnev**
###### Motivation for this change

`tree-from-tags` is a tool for organising tagged audio into a tree, commonly used with `bongo`, a buffer-oriented media player for `emacs`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @dbrock

### CODE REVIEW

**Comment by joachifm** [comment]
@GrahamcOfBorg build tree-from-tags

File: pkgs/applications/audio/tree-from-tags/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    name = "tree-from-tags-${version}-gems";
+    inherit ruby;
+    gemdir  = ./.;
+  };
+in stdenv.mkDerivation {
+  name = "tree-from-tags-${version}";
+  src = fetchFromGitHub {
+    owner  = "dbrock";
+    repo   = "bongo";
+    rev    = "3d246be1e8d14865f5253567ab8fee5d4e9c470c";
```
[the remark this answers is not in the document]
Comment (livnev) [reply pkgs/applications/audio/tree-from-tags/default.nix]: Done, and force pushed.

**Comment by joachifm** [comment]
@GrahamcOfBorg build tree-from-tags

**Comment by joachifm** [comment]
Thank you

**Pull request merged**